packages/dev/parcel-transformer-docs/DocsTransformer.js

```diff
@@ -508,6 +527,20 @@ module.exports = new Transformer({
         });
       }
 
+      if (path.isTSModuleDeclaration()) {
```

Comment:
this comes from MessageFormatter -> intl-messageformat -> formatters.d.ts which declares a global namespace FormatjsIntl which our docs transformer doesn't know how to handle. I'm not sure how to handle it either, so for now, return node